integration/cmd/nodes2way-builder/store.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"time"
+
+	"github.com/Telenav/osrm-backend/integration/util/waysnodes"
+	"github.com/Telenav/osrm-backend/integration/util/waysnodes/nodes2wayblotdb"
+	"github.com/golang/glog"
+)
+
+func newStore(in <-chan *waysnodes.WayNodes, out string) (err error) {
+	startTime := time.Now()
+
+	db, err := nodes2wayblotdb.Open(out, false)
+	if err != nil {
+		return
+	}
+	defer func() {
+		err = db.Close()
+	}()
+
+	var inCount, succeedCount int
+	for {
+		wayNodes, ok := <-in
+		if !ok {
+			break
+		}
+		inCount++
+
+		if err := db.Write(wayNodes.WayID, wayNodes.NodeIDs); err != nil {
+			if glog.V(3) { // avoid affect performance by verbose log
+				glog.Infof("Update %+v into db failed, err: %v", wayNodes, err)
+			}
```

Comment:
Just confirm: `glog.V(3).Infof` should be able to achieve the same goal.  I saw the document mentioned both.

```go
//	if glog.V(2) {
//		glog.Info("Starting transaction...")
//	}
//
//	glog.V(2).Infoln("Processed", nItems, "elements")
```

Reply:
Functionally they're the same. The difference is that the first one is cheaper if logging is off because it does not evaluate its arguments, and the seconder one is shorter. See more in https://godoc.org/github.com/golang/glog#V.